src/syrupy/extensions/amber.py

```diff
@@ -56,12 +57,12 @@ def read_file(cls, filepath: str) -> "SnapshotFossil":
         indent_len = len(cls._indent)
         snapshot_fossil = SnapshotFossil(location=filepath)
         try:
-            with open(filepath, "r") as f:
+            with open(filepath, "r", newline="") as f:
                 test_name = None
                 snapshot_data = ""
                 for line in f:
                     if line.startswith(cls._marker_name):
-                        test_name = line[name_marker_len:-1].strip(" \n")
+                        test_name = line[name_marker_len:-1].strip(" \r\n")
```

Comment:
We should call out that amber does not preserve trailing newlines... nb. but then it's kinda obvious from the output and I doubt a lot of people would care

Reply:
It does preserve trailing newlines for serialized content. This is for the test name.